.gitignore

```diff
@@ -0,0 +1,29 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.eslintrc.json
+/package-lock.json
+/package.json
```

Comment:
NEVER put package.json to the .gitignore without it it's impossible for other devs in team to run the project

Reply:
I especially added package.json to .gitignore because that file have a different command and dependencies like a babel and I thought probably other students won't use babel and other dependencies.